.circleci/config.yml

```diff
@@ -34,6 +34,14 @@ defs:
       # because macOS machine comes with node v8.6.0
       command: curl "https://nodejs.org/dist/v10.10.0/node-v10.10.0.pkg" > "$HOME/Downloads/node-latest.pkg" && sudo installer -store -pkg "$HOME/Downloads/node-latest.pkg" -target "/"
 
+    step-install-arduino-cli-on-mac: &step-install-arduino-cli-on-mac
+      name: Install arduino-cli
+      command: mkdir "$HOME/Downloads/arduino-cli" && curl "https://downloads.arduino.cc/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx.zip" > "$HOME/Downloads/arduino-cli/arduino-cli.zip" && unzip "$HOME/Downloads/arduino-cli/arduino-cli.zip" -d "$HOME/Downloads/arduino-cli" && mv "$HOME/Downloads/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx" "$HOME/arduino-cli"
```

Comment:
I suggest change this:
```
mkdir "$HOME/Downloads/arduino-cli" && curl "https://downloads.arduino.cc/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx.zip" > "$HOME/Downloads/arduino-cli/arduino-cli.zip" && unzip "$HOME/Downloads/arduino-cli/arduino-cli.zip" -d "$HOME/Downloads/arduino-cli" && mv "$HOME/Downloads/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx" "$HOME/arduino-cli"
```
into this:
```
curl -s --create-dirs -o "$HOME/Downloads/arduino-cli/arduino-cli.zip" "https://downloads.arduino.cc/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx.zip" && unzip "$HOME/Downloads/arduino-cli/arduino-cli.zip" -d "$HOME/Downloads/arduino-cli" && mv "$HOME/Downloads/arduino-cli/arduino-cli-0.2.2-alpha.preview-osx" "$HOME/arduino-cli"
```

Reply:
May I also suggest line wrapping at arg boundaries if they do not fit 80 chars of Yaml?